cocos/animation/marionette/animation-graph-animation-clip-binding.ts

```diff
@@ -153,6 +160,24 @@ class PoseScaleBinding extends PoseBindingBase implements PoseBinding<Vec3> {
     }
 }
 
+class AuxiliaryCurveBinding implements PoseBinding<number> {
+    constructor (private _handle: AuxiliaryCurveHandle) {
+
+    }
+
+    public destroy (): void {
+        this._handle.destroy();
+    }
+
+    public setValue (value: number, pose: Pose): void {
+        pose.metaValues[this._handle.index] = value;
```

Comment:
`metaValues` maybe needs a new name related to `AnxuliarayCuve`

Reply:
Solved in https://github.com/cocos/cocos-engine/pull/14659/commits/29a9d7445661dffcd5ebdd44173d6628f25a6dfd .